tests/fuzz/regression_test.go

```diff
@@ -40,7 +40,7 @@ var brokenCases = map[string]string{
 	"6087702507290624": "https://github.com/go-yaml/yaml/issues/768",
 }
 
-func runRegressionTest(t *testing.T, name string, fuzz func(data []byte) int) {
+func runRegressionTest(t *testing.T, name string, fuzz func(data []byte) int, expectBadCasesToFail bool) {
```

Comment:
I would prefer to remove the changes to this file and remove the 2 current brokenCases. The intent of broken cases is that this _will_ panic for the given test. So to have `expectBadCasesToFail=false` doesn't really make sense.

Also, contrary to what I mentioned in chat, the broken cases do actually only run for a specific test case (determined by testdata folder). So we do not lose coverage at all by removing those 2 from brokenCases